h264/cabac.go

```diff
@@ -1,6 +1,9 @@
 package h264
 
-import "github.com/pkg/errors"
+import (
+	"github.com/ausocean/h264decode/h264/bits"
+	"github.com/pkg/errors"
```

Comment:
See previous comments about introducing this as a dep. Note that mrmod's code does not import it.